Source/control.cpp

```diff
@@ -567,7 +568,7 @@ void InitControlPan()
 		buttonEnabled = false;
 	chrbtnactive = false;
 	pDurIcons = LoadCel("Items\\DurIcons.CEL", 32);
-	strcpy(infostr, "");
+	InfoString = "";

```

Comment:
There's a few places you can use `string::clear()` instead of copy-assigning from an empty string. If it weren't for the calls to fmt I'd be tempted to use `string_view` for `InfoString` and leave these assignments as is since it works better with a view.
```suggestion
	InfoString.clear();
```